### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by ain** [fatfreecrm/fat_free_crm#405]
Would be nice to have an option to send a newsletter to entities like Accounts and/or Leads for instance, all at once.

**Comment by steveyken** [comment]
Yeah, there have been various attempts at mailchimp integration but this sounds ripe for a plugin that can use Griddler and the new ActiveJob API (rails 4.2) to send messages.

**Comment by johnnyshields** [comment]
@steveyken @CloCkWeRX I think the mailing list (newsletters, marketing mails, etc) functionality should be brought into the core of FFCRM and the MailChimp (or other service) plugins should be relatively simple adapters.

**Comment by CloCkWeRX** [comment]
What does the most minimal version of this look like?

https://github.com/mailchimp/mailchimp-marketing-ruby?tab=readme-ov-file#api-endpoints

- Product catalog?
- Campaign has many emails?
- etc

Its... very broad at the moment.
